### PR TITLE
Update hour symbol to `h`

### LINKED ISSUE #719: "h" should be the abbreviation of hours
"hr" is used as an abbreviation for hours, but this is not according to standard (see https://www.bipm.org/en/CGPM/db/9/7/), and much less common than "h" worldwide.

Here "h" is used for the Planck constant, which I think has much less use than the hours symbol. Both "h" and "hr" should be used for hours, to avoid misunderstanding. Planck should be called something else.

### PR DESCRIPTION
**Pull request by jules-ch**
- Replace h symbol for planck constant to  `ℎ`
- Add breaking change in CHANGES

- [x] Closes #719
- [x] Executed ``pre-commit run --all-files`` with no errors
- [x] The change is fully covered by automated unit tests
- [x] Documented in docs/ as appropriate
- [x] Added an entry to the CHANGES file

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -19,6 +19,8 @@ Pint Changelog
 
 ### Breaking Changes
 
+- Update hour default symobl to `h`. (Issue #719)
```
Comment (dopplershift) [inline CHANGES:22]: "symobl" -> "symbol"

**Comment by jules-ch** [comment]
@hgrecco tell me if that's ok with you.

**Comment by hgrecco** [comment]
I agree with this change. But I think with this change!

**Comment by hgrecco** [comment]
Sorry, I was very tired that day :-D  I was saying that it should be documented as breaking change and then I noticed that was the case!

**Comment by jules-ch** [comment]
Alright I'm gonna merge this then.

**Pull request merged**